v0/destinations/am/transform.js

```diff
@@ -206,6 +207,8 @@ function responseBuilderSimple(
       endpoint = ENDPOINT;
       // event_type for identify event is $identify
       rawPayload.event_type = EventType.IDENTIFY_AM;
+      rawPayload.country = get(message, "context.location.country");
+      rawPayload.city = get(message, "context.location.city");
```

Comment:
Some of the mapping changes made here can be driven from the mapping config JSON files

Reply:
changed to use default mapping config JSON